detect/fixeddetector.py

```diff
@@ -0,0 +1,29 @@
+import pandas as pd
+from .detector import Detector
+from dataset import Table
+
+class FixedDetector(Detector):
+    def __init__(self, name, src, fpath=None, 
```

Comment:
Can we modify the constructor to accept a csv file path and that's it for now? we can add a `Table` parameter later.

Reply:
Removed the Table instance and replaced it with a csv read directly, changed the constructor to reflect.